arm/authorization/version.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	major = "5"
+	major = "6"
```

Comment:
Just curious, do you guys track the breaking changes? if so it'd be great to put it somewhere with the release (in Changelog or breakingchanges file)